docs/actions.md

```diff
@@ -18,42 +18,47 @@ An action is comprised with the following properties:
-| `url`         | string                                       | ✓         | Path name to use for the `POST` request                                                                                                                                                                                                             |
+| `context`     | enum('provisioner', 'worker-type', 'worker') | ✓         | Actions have a "context" that is one of `provisioner`, `worker-type`, or `worker`, indicating which it applies to.                                                                                                                                  |
+| `url`         | string                                       | ✓         | URL to use for the `POST` request.                                                                                                                                                                                                                  |
 | `description` | string                                       | ✓         | A human readable string describing the action, such as what it does, how it does it, what it is useful for. This string is to be render as markdown, allowing for bullet points, links and other simple formatting to explain what the action does. |
 
-
 ### Context
 Actions have a "context" that is one of `provisioner`, `worker-type`, or `worker`, indicating which it applies to. `context`
-is used to construct the query string of the `POST` request.
+is used by the front-end to know where to display the action.
 
-| `context`     | query string                                                     |
-|---------------|------------------------------------------------------------------|
-| `provisioner` | `?provisionerId=...`                                             |
-| `worker-type` | `?provisionerId=...&workerType=...`                              |
-| `worker`      | `?provisionerId=...&workerType=...&workerGroup=...&workerId=...` |
+| `context`     | Page displayed        |
+|---------------|-----------------------|
+| `provisioner` | Worker Types Explorer |
+| `worker-type` | Workers Explorer      |
+| `worker`      | Worker Explorer       |
 
 ## How to trigger an action
-To trigger an action, make a `POST` request to a URL constructed from the url field with query
-arguments added based on the `context`.
+To trigger an action, use the action's `url` property to make a `POST` request to it.
+Depending on the action's `context`, substitute the following parameters in the `url`:
+
+| `context`   | Path parameters                                      |
+|-------------|------------------------------------------------------|
+| provisioner | :provisionerId                                       |
```

Comment:
Can we use `<provisionerId>`  like we do in `entries[i].route` in `reference.json` files, see docs:
https://docs.taskcluster.net/manual/design/apis/reference-format